CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-
+- Add: max time detection for non singal rules to not see an entity as silent due to is death (#827)
+- Add: env var PERSEO_NON_SIGNAL_MAX_TIME_DETECTION to set global nonSignalMaxTimeDetection (#827)
```

Comment:
Issue reference should be #824?

Reply:
Done in https://github.com/telefonicaid/perseo-fe/pull/827/commits/3622af535a9bc713813f385f9d380989bbdcee9c